### ISSUE TITLE
Compiling testing function   doesn't work

### ISSUE THREAD

**Issue by kealennieh** [chrischoy/3D-R2N2#11]
Dear author:
    sorry to bother you. Here is my question. After I change the training dataset, the training process cannot operate successfully. It always loops in "Compiling testing function", as follows:

lib/data_io.py: model paths from ./experiments/dataset/shapenet_1000.json
lib/data_io.py: model paths from ./experiments/dataset/shapenet_1000.json
[INFO/ReconstructionDataProcess-1] child process calling self.run()
Set the learning rate to 0.000100.
[INFO/ReconstructionDataProcess-2] child process calling self.run()
Compiling training function
Exiting
[INFO/ReconstructionDataProcess-2] process shutting down
2017-06-27 09:06:57.187543 Iter: 0 Loss: 0.502873
[INFO/ReconstructionDataProcess-2] process exiting with exitcode 0
Compiling testing function

I attempted a lot of times, but failed at the same place. I don't know the reason, since I just change the training dataset. Thanks in advance.

**Comment by chrischoy** [comment]
It is possible that the json file is empty or the process cannot load it properly and the network fails. Try to make an isolated test function like https://github.com/chrischoy/3D-R2N2/blob/master/lib/data_process.py#L216 and see if you can load the dataset properly first.
